### ISSUE TITLE
Question of Overfitting!

### ISSUE THREAD

**Issue by lawsonxwl** [magicleap/SuperGluePretrainedNetwork#98]
Thanks for solid work!
I use 80 scenes in megadepth to train the superglue model
but only after 5 epoch,the model converges and the validation loss explodes after epoch 5.
![image](https://user-images.githubusercontent.com/52605055/129537544-fbfb7cf9-6033-410b-a079-ea7e196481f4.png)

**Comment by Skydes** [comment]
Sorry for the late reply. 80 scenes is definitely not sufficient. SuperGlue is a large-capacity models and therefore needs a lot more training data than previous works.
